lib/platform/gitea/gitea-helper.spec.ts

```diff
@@ -773,15 +774,27 @@ describe('platform/gitea/gitea-helper', () => {
         'warning',
         'failure',
         'error',
+        'failure',
+        'warning',
+        'pending',
+        'success',
+        'unknown',
       ];
 
       const commitStatuses: ght.CommitStatus[] = [
         { ...mockCommitStatus, status: 'unknown' },
       ];
+      const now = new Date();
```

Comment:
Use fixed timestamps for testing, so you can simply use snapshots or equals. Also easier to reproduce